shipstation_integration/shipments.py

```diff
@@ -158,3 +184,57 @@ def create_delivery_note(shipment, sales_invoice):
 	dn.submit()
 	frappe.db.commit()
 	return dn
+
+
+def create_shipment(
+	shipment: "ShipStationOrder",
+	delivery_note: "DeliveryNote",
+	store: "ShipstationStore",
+):
+	shipment_doc: "Shipment" = make_shipment(delivery_note.name)
+	shipment_doc.update(
+		{
+			# todo: change description of content to shipment items.
+			"shipment_id": shipment.shipment_id,
+			"pickup_date": shipment.create_date,
+			"carrier": shipment.carrier_code,
+			"carrier_service": shipment.service_code,
+			"awb_number": shipment.tracking_number,
+			"service_provider": "Shipstation",
+			"incoterm": "DAP (Delivered At Place)",
+			"shipstation_store_name": store.store_name,
+			"shipstation_order_id": shipment.order_id,
+			"marketplace": store.marketplace_name,
+			"marketplace_order_id": shipment.order_number,
+		}
+	)
+
+
+	if shipment.shipment_items:
```

Comment:
What child table is this? I can't see it anywhere locally.

Reply:
This is object from api response.

Reply:
Oh, I was comparing it to the wrong object lol. This is fine.